### ISSUE TITLE
Blank screen when viewing Jobs or Tickets

### ISSUE THREAD

**Issue by githubbob42** [githubbob42/mingle2github2#3875]
**[Mingle Card: 4202](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/4202)**
Steps to reproduce
------------------

```
| | |
|-|-|
|**Version #**|
| |
|**OS**|
| |
|**Browser**|
| |
|**Username**|
|
   

mdeville@liquidframeworks.combes.bes1q16

   |
|**Password**|
|fieldfx1|
  
 

```

1.  Login to mobile
2.  Goto Jobs (or Tickets) list
3.  click on a Job (or Ticket)
4.  blanks screen

Expected result
---------------

_Object should render_

Actual Result
-------------

_Blank screen_

_Analysis_
----------

We are getting a namespace collision with Mappings.getSObjectName and it’s returning the wrong object name.  

Basic Energy has both FieldFX and BMC ServiceDesk installed in their Org.  Both apps have a Status\_\_c object (both namespaced) but when Mappings.getSObjectName finds more than one object, it optimistically return the first one and BMCServiceDesk\_\_Status\_\_c comes before FX5\_\_Status\_\_c, so we get the wrong object name returned.

Then when the WorkfowPicker tries to load the Status object or when the page tries to determine if the current Status is readonly, it tries to read the “BMCServiceDesk\_\_Status\_\_c” table from the sobjects database and that of course throws an error.

Related Cards
-------------

PRs
---

[https://github.com/Liquidframeworks/alpine-mobile/pull/1906](https://github.com/Liquidframeworks/alpine-mobile/pull/1906)

[https://github.com/Liquidframeworks/alpine-mobile/pull/1873](https://github.com/Liquidframeworks/alpine-mobile/pull/1873)

[https://github.com/Liquidframeworks/alpine-mobile/pull/1910](https://github.com/Liquidframeworks/alpine-mobile/pull/1910)

Test Plan
---------

*   Duplicate Objects – Namespace Collision

[qaadmin@test4.com](mailto:qaadmin@test4.com)  
[qafield@test4.com](mailto:qafield@test4.com)

computer45

**Comment by githubbob42** [comment]
**Pull Request #1873**

**Comment by githubbob42** [comment]
Matthew Todd : 07/01/2016 10:09 CDT
Passed PR Build: 4.0.1.pr1873 Admin User: qaadmin@test4.com Field User: qafield@test4.com FieldFX URL: https://dev.fieldfx.com Hardware: Windows 10 Pro Desktop (Chrome) This has passed testing on with build 4.0.1.pr1873. With the help of Roy to create a new package that contains FX0\_\_Status\_\_c, I was able to install this and recreate the issue on my ORG. After the failure was evident, I pointed to Bob’s fix and all is working. Extra time was spent around Quotes, editing, creating new, and all of that worked, too.
